package.json

```diff
@@ -54,14 +54,28 @@
     "packages": [
       "design-system",
       "generators/*",
-      "packages/**/*",
+      "packages/*",
+      "packages/components/*",
+      "packages/components/buttons/*",
+      "packages/components/fields/*",
+      "packages/components/inputs/*",
+      "packages/components/spacings/*",
```

Comment:
The changes here are required (it's the only way I could find to make things work) as issues with manypkg vs preconstruct started to appear when I added the new entrypoint.

For instance, it was asking me to put a `name` field to the `package.json` of the entrypoint. But doing that caused another incompatible error.

So instead of specifying a `**/*` glob to include subfolders, I'm explicitly providing the path to nested packages.